src/components/AttachmentPicker/index.native.js

```diff
@@ -117,17 +118,22 @@ class AttachmentPicker extends Component {
       * Handles the image/document picker result and
       * sends the selected attachment to the caller (parent component)
       *
-      * @param {ImagePickerResponse|DocumentPickerResponse} attachment
+      * @param {Array<ImagePickerResponse|DocumentPickerResponse>} attachments
       */
-    pickAttachment(attachment) {
-        if (attachment) {
-            if (attachment.width === -1 || attachment.height === -1) {
-                this.showImageCorruptionAlert();
-                return;
-            }
-            const result = getDataForUpload(attachment);
-            this.completeAttachmentSelection(result);
+    pickAttachment(attachments = []) {
+        if (attachments.length === 0) {
+            return;
         }
+
+        const fileData = _.first(attachments);
+
+        if (fileData.width === -1 || fileData.height === -1) {
+            this.showImageCorruptionAlert();
+            return;
+        }
+
+        const result = getDataForUpload(fileData);
+        this.completeAttachmentSelection(result);
```

Comment:
The updated document picker responds with an array. The image picker would also respond with array that's why this was changed to accept an array value